challenges/objects-arrays.js

```diff
@@ -80,9 +114,16 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
-const lowerCase = [];
-console.log(lowerCase); 
+const lowerCase = []
+zooAnimals.map(function(element) {
+  lowerCase.push(element.animal_name.toLowerCase());
+})
+console.log(lowerCase);
 
 // The zoos need to know their total animal population across the United States.  Add up all the population numbers from all the zoos using the .reduce() method.
 const populationTotal = [];
-console.log(populationTotal);
\ No newline at end of file
+zooAnimals.forEach(function(element) {
+  populationTotal.push(element.population);
+})
+const reducer = (accumulator, currentValue) => accumulator + currentValue;
+console.log(populationTotal.reduce(reducer));
```

Comment:
nice! you also could have done this (which would have been a bit faster due to only looping once instead of twice) :
```
const answer = zooAnimals.reduce((accumulator, animal) => accumulator + animal.population, 0);
console.log(answer); 
```